secator/template.py

```
@@ -1,30 +1,29 @@
-from secator.rich import console
-from secator.utils import convert_functions_to_strings, debug
 from secator.output_types import Error
-
-TEMPLATES = []
+from secator.rich import console
+from secator.utils import convert_functions_to_strings
 
 
 class TemplateLoader(DotMap):
 
 	def __init__(self, input={}, name=None, **kwargs):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)